scripts/swupd-performance/data_processing_script.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python3
+"""
+This module is created to clean, parse output of swupd update service
+and create a CSV file
+"""
+
+import argparse
+import os
+import json
+#from collections import OrderedDict
+import csv
+import logging
+from datetime import datetime
```

Comment:
what linter are you running? I see you have the import list unordered, that is usually caught by linters, also you are mixing "import ##" with "from ## import ##", if I recall correctly they need to be ordered separately,

Reply:
So I ran pylint and the file_name. This pylint came with one of bundles

Reply:
I ran autopep on the code, so these should be ordered right

Reply:
@castulo I run pylint and I found no errors. Probably I'm doing something wrong. I just installed pylint from pip and run:

    pytlint stript_name.py

Is there any extra parameter that you use?

Reply:
I ran like @otaviobp

Reply:
yeah, let me take a look at this, ill run it locally first and will tell you my findings

Reply:
pylint is probably the most powerful of the python linters but it is also highly configurable and not that user friendly if you are not very familiar with python. For a big python project I would recommend spending time configuring pylint to get it exactly as we expect the code to be. But doing that for only a few scripts seems like too much overhead.
So for these python scripts I'd recommend running these two linters:

- `pycodestyle` (pip install pycodestyle)
- `flake8` (pip install flake8)

And just run:
```
$ pycodestyle data_processing_script.py
$ flake8 data_processing_script.py --statistics
```
Those should enforce compliance with at least PEP8 and few other code style good practices.

Reply:
I am okay running these two..especially flake8 is really good. I resolved all but one using them. Looks like that one if tough.

Reply:
what is it about?

Reply:
Line more than 80 char

Reply:
I just downloaded the patch and ran flake8 and I get all these:
```
1     D200 One-line docstring should fit on one line with quotes
1     D202 No blank lines allowed after function docstring
11    D205 1 blank line required between summary line and description
12    D400 First line should end with a period
11    D401 First line should be in imperative mood
6     D413 Missing blank line after last section
1     E501 line too long (83 > 79 characters)
```

Reply:
I ran flake8 just like you did...did you run with flake8 data_processing_script.py --statistics?

Reply:
I don't understand how you don't get the warnings, I can see the style violations in the code, for example, looking at the first warning I get:
`1     D200 One-line docstring should fit on one line with quotes`

I can clearly see what is triggering this in line 313:
```
"""Main function
     """
```

This should be:
`"""Main function"""`

It's odd you don't see those style violations.

Reply:
So look like there were flake docstring linters which missed this. I fixed all those pointed by them Except one that i pointed out earlier.